app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
```

Comment:
- [ ] [OPTIONAL] I strongly recommend that you remove the foreign keys on the comments and likes. According to the ERD, the User table only has a foreign on the Post.